### ISSUE TITLE
Questions/thoughts on "Physical quantities..."

### ISSUE THREAD

**Issue by davidlmobley** [kofkeLab/tutorials#4]
On the physical quantities document, I had these thoughts/questions on a quick read through:
- Do we need to define extensive/intensive more directly? (Perhaps with the convenience of GitHub we should actually start a glossary of definitions in a separate document and just make these hyperlinks to the glossary?)
- I'm proposing (in a separate PR) some very basic edits that hopefully are non-objectionable, but do you have a preferred way to get comments of the "Maybe this part could be more clear because of XYZ..." or "Would it make sense to address ABC here...?" sort? (Concrete example: p2 mentions torques being needed to advance an MD simulation, but I'm not aware of separate analysis of torques; I believe most integrators simply advance coordinates based on forces/velocities/positions? What am I missing?)
- Should I reformat Figure 1 as a table? 
- In the section on dimensions/and units, perhaps we can recommend that user-facing components of a program use a programming library which accommodates and handles units so users can, say, provide a pressure in atm or, say, pascals and still have the intended results? Or at least we could suggest this as an option. 
- Any thoughts on reference handling? BibTeX OK? Separate references for each "chapter"? (comes up on p5 of this document)

**Comment by kofke** [comment]
> Do we need to define extensive/intensive more directly? 

Yes, certainly a glossary could be useful.  It's another layer of effort though.  It might make sense to make a few entries and see how it works with the text.

> do you have a preferred way to get comments 

No, I haven't worked this way before, so I don't have preconceived notions of how to communicate specific comments.  I'm open to suggestions.  

>I believe most integrators simply advance coordinates based on forces/velocities/positions

Yes, you're probably right about that.  I don't remember what I had in mind when that was written.  Perhaps I had been working on rigid particles (e.g., ellipsoids) then, or rough spheres; these are uncommon cases though.

> Should I reformat Figure 1 as a table?

Yes, I thought of doing that but didn't get to it.

> recommend that user-facing components of a program use a programming library 

Sounds good.

> Any thoughts on reference handling?

Yes to BibTeX!  Probably it would be easier to manage references by separating them with the chapters. I prefer numbering them (rather than author-year), which would be difficult to maintain across several documents (though maybe it's no problem for LaTeX?).

**Comment by davidlmobley** [comment]
OK, this all sounds good.  I'll try to work on these issues. 

I think for the moment I may just make the glossary a markdown (basically nicely-formatted text) file to make it easy to link to specific sections of it from the individual documents, and we can decide later if this is the right way to do things.

**Comment by davidlmobley** [comment]
I'm adding the beginnings of a very rudimentary glossary in #7 . Probably we'll want the chapter contents to provide links to the glossary -- not necessarily when terms are used, but just to make obvious that it's available, but this should come later after the glossary has more in it.
